hashicorp/boundary-ui
# fix: 🐛 code block error in console when value is empty on save

# Description

Fix hds code block error manifesting itself when model is saving and code block is empty.




## Screenshots (if appropriate)
Error msg:
![image](https://github.com/user-attachments/assets/14a48ffc-e310-4178-9e0b-63e82d8011b3)

After fix:

https://github.com/user-attachments/assets/00fc504d-228b-4a59-8ae3-4eb89daa74c1


## How to Test

1. Go to new storage bucket form
2. Try to save without a value in the code editor
3. Make sure no error is in the console.

## Checklist
